theme_5/scss/_inputs.scss

```diff
@@ -0,0 +1,33 @@
+// Input
+.text-container {
+	border-bottom: 1px solid $cerise-red;
+	margin: 0 15px 10px;
+	.text-box {
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented with spaces, not tabs